UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs

```diff
@@ -76,8 +76,23 @@ private bool CanAttatchToTarget(Matrix matrix, RegisterTile tile)
 
 		public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
 		{
-			if (DefaultWillInteract.Default(interaction, side) == false
-			    || isArmed == true || pickupable.ItemSlot == null || interaction.Intent != Intent.Harm) return false;
+			if (DefaultWillInteract.Default(interaction, side) == false || pickupable.ItemSlot == null) return false;
+
+			// Why do we prevent mounting if it is armed?
```

Comment:
@MaxIsJoe

Reply:
ayo dont remove the harm intent, its intentional design

Reply:
actually nevermind apparently he added a message to tell the player about it, thats nice.